src/platform/windows/events_loop.rs

```diff
@@ -178,6 +195,13 @@ impl EventsLoop {
         }
     }
 
+    fn sync_with_thread(&self) {
```

Comment:
Same here. Even though it seems to violate DRY, I'd prefer is this was just copy-pasted in the two places where this is relevant. Having a function that must only be called under precise circumstances is never good IMO.